dynamo/vectorfield/scVectorField.py

```diff
@@ -427,9 +432,9 @@ def SparseVFC(
     tmp_X, uid = np.unique(X, axis=0, return_index=True)  # return unique rows
     M = min(M, tmp_X.shape[0])
     if velocity_based_sampling:
-        if verbose > 1:
-            print("Sampling control points based on data velocity magnitude...")
-        idx = sample_by_velocity(Y[uid], M, seed=seed)
+        np.random.seed(seed)
```

Comment:
IMPORTANT: PLEASE pass seed to sample_by_velocity! Otherwise a new seed will be used in `sample_by_velocity`. This ensure you can get different results for sparseVFC with different seeds

Reply:
You are right about it. This should be an issue generated when I merge things.